PFR/Mathlib/Data/Fintype/Basic.lean

```diff
@@ -0,0 +1,27 @@
+import Mathlib.Data.Fintype.Basic
+
+namespace Fintype
+
+variable {α : Type*} [Fintype α]
+
+def elem (i : α) : Finset.univ (α := α) := ⟨i, Fintype.complete i⟩
+
+theorem elem_bijective : Function.Bijective Fintype.elem (α := α) :=
+  ⟨ fun _ _ ↦ Subtype.mk_eq_mk.mp, Function.LeftInverse.surjective (congrFun rfl) ⟩
```

Comment:
Can you do without introducing this? We don't want this in mathlib.

Reply:
I was able to do it with no lemmas. So all of the following can removed as well (but let me know if any are worth keeping):

```lean
def toFinset_apply {s : Set α} [Fintype s] (i : s) : s.toFinset
theorem toFinset_bijective {α : Type*} (s : Set α) [Fintype s] : Function.Bijective (Set.toFinset_apply (s := s))
theorem bijective_onto_image {α β : Type*} {f : α → β} {S : Set α} (hf : Function.Injective f) : Function.Bijective (Set.imageFactorization f S)
```

Reply:
I think those are all worth removing.